src/vue/modules/create-asset-sale/store/index.js

```diff
@@ -0,0 +1,40 @@
+import { types } from './types'
+import { api } from '../_api'
+import { AssetRecord } from '../wrappers/asset.record'
+//
+// const HORIZON_VERSION_PREFIX = 'v3'
+
+export const state = {
```

Comment:
Empty state and mutations are disallowed

Reply:
ok